app/uk/gov/hmrc/excisemovementcontrolsystemapi/models/messages/IEMessage.scala

```diff
@@ -19,8 +19,15 @@ package uk.gov.hmrc.excisemovementcontrolsystemapi.models.messages
 import scala.xml.NodeSeq
 
 trait IEMessage {
-  def localReferenceNumber: Option[String]
-  def getType: String
+  // def not val as will be evaluated straight away
+//  def localReferenceNumber: Option[String]
+//  def consignorId: Option[String]
+  def consigneeId: Option[String]
+  def administrativeReferenceCode: Option[String]
+  def messageType: String
   def toXml: NodeSeq
+  def getErns: Set[String] //= Set(consignorId, consigneeId).flatten
```

Comment:
Can this commented out code be removed please?

Reply:
done